qa/L0_shared_memory/shared_memory_test.py

```diff
@@ -295,7 +317,7 @@ def test_too_big_shm(self):
     def test_mixed_raw_shm(self):
         # Mix of shared memory and RAW inputs
         error_msg = []
-        shm_handles = self._configure_sever()
+        shm_handles = self._configure_sever(shm_byte_size=256)
```

Comment:
My PR will have some conflicts with this PR, but you should be able to use the `create_byte_size` and `register_byte_size` args in the same way. Let me know if you're good with it: https://github.com/triton-inference-server/server/pull/7093

Reply:
Francesco's PR will also likely have some merge conflicts: https://github.com/triton-inference-server/server/pull/7048